app/src/main/java/com/android/feedme/ui/camera/Camera.kt

```diff
@@ -82,38 +67,27 @@ import com.android.feedme.ui.theme.CameraButtonsBackground
  * images and viewing them in a gallery. Utilizes CameraX for camera operations and Jetpack Compose
  * for the UI components.
  */
-@OptIn(ExperimentalMaterial3Api::class)
+@OptIn(ExperimentalMaterial3Api::class, ExperimentalCoroutinesApi::class)
```

Comment:
What does this add-on do ?

Reply:
It was a remnant of something I tried using coroutines to wait for the result of the photo before navigating to the second screen, before settling with using Booleans. I will remove it :)